docs/kratos/session-management/15_check-session.mdx

```diff
@@ -1,109 +0,0 @@
----
```

Comment:
I had to move this file out of this directory because the directory is used for auto-generated TOC.